<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">*{
border: none;
}

#widget{
background-color: rgb(29, 35, 50)
}

QTabBar::tab {
background-color: rgb(227, 227, 227);
width: 160px;
height: 55px;
padding-left: 10px;
padding-right : 10px;
border-top-left-radius: 5px;
border-top-right-radius: 5px; 
font: 63 18pt &quot;Bahnschrift SemiBold Condensed&quot;;
margin-right: 5px
}

#tab{
background-color: rgb(227, 227, 227)
}

#enregistrerButton{
background-color: rgb(30, 255, 30);
}

#annulerButton{
background-color : rgb(255, 29, 29)
}

#enregistrerButton, #annulerButton{
color: white;
padding: 8px;
}

#listWidget{
border: 1px solid black;
background-color: white;
color: rgb(29, 35, 50);
border-radius: 5px
}

#addCategoryButton, #removeCategoryButton{
background-color: rgb(231, 231, 231);
padding: 5px;
border: 1px solid rgb(29, 35, 50);
border-radius: 5px
}

QPushButton{
border-radius: 5px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="icon">
          <iconset>
           <normaloff>../pictures/requisition.png</normaloff>../pictures/requisition.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Categories</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QListWidget" name="listWidget">
            <property name="font">
             <font>
              <family>Segoe Print</family>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Braclets</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Rings</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ear Rings</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item alignment="Qt::AlignLeft">
              <widget class="QWidget" name="widget_3" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QPushButton" name="addCategoryButton">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../interface_images.qrc">
                    <normaloff>:/newPrefix/pictures/plus.png</normaloff>:/newPrefix/pictures/plus.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="removeCategoryButton">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../interface_images.qrc">
                    <normaloff>:/newPrefix/pictures/minus-sign.png</normaloff>:/newPrefix/pictures/minus-sign.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item alignment="Qt::AlignRight">
              <widget class="QWidget" name="widget_4" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QPushButton" name="enregistrerButton">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Enregistrer</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="annulerButton">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Annuler</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../interface_images.qrc"/>
 </resources>
 <connections/>
</ui>
